Source/Core/Core/HW/WiimoteReal/IOWin.cpp

```diff
@@ -395,18 +500,21 @@ void WiimoteScanner::CheckDeviceType(std::basic_string<TCHAR> &devicepath, bool
 		u8 const disable_enc_pt2_report[MAX_PAYLOAD] = {WM_SET_REPORT | WM_BT_OUTPUT, WM_WRITE_DATA, 0x04, 0xa4, 0x00, 0xfb, 0x01, 0x00};
 
 		CheckDeviceType_Write(dev_handle,
-		                      disable_enc_pt1_report,
-		                      sizeof(disable_enc_pt1_report),
-		                      1);
+							  write_method,
+							  disable_enc_pt1_report,
+							  sizeof(disable_enc_pt1_report),
+							  1);
```

Comment:
Either align this correctly (two tabs, the rest spaces) or don't align it at all (no spaces, something like three or four tabs). I'd prefer the former.